index.js

```diff
@@ -1,12 +1,14 @@
+/* eslint-disable no-var */
+
```

Comment:
Since we're changing this already, let's switch to `const` :)

Reply:
Ok. I just thought es5 support is still needed.

Reply:
This will be a breaking change so we can just stop supporting it